.github/workflows/pr-rules.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.15
+          go-version: 1.16
```

Comment:
I wonder if we should just eliminate this setup since Ubuntu 20.04 Github Actions runner includes this by default anyway: https://github.com/actions/virtual-environments/blob/ubuntu20/20210219.1/images/linux/Ubuntu2004-README.md#go

We actually current don't have it for tracee-ebpf action: https://github.com/aquasecurity/tracee/blob/main/.github/workflows/pr-ebpf.yaml#L17-L25

Reply:
we don't have it in tracee-ebpf because it runs in a container.
personally I think it's better to be specific which version we are using to avoid future bugs